src/index.spec.js

```diff
@@ -580,16 +580,16 @@ describe('Dropzone', () => {
         dataTransfer: { files: images.concat(files) }
       })
       const rejected = dropSpy.firstCall.args[1]
-      expect(rejected.length).toEqual(2)
+      expect(rejected.length).toBeGreaterThan(1)
     })
 
-    it('should allow single files to be dropped if multiple is false', () => {
+    it('should reject single files on multiple drop if multiple is false', () => {
```

Comment:
Is there also a test where you test that all files would end up in `rejected`?

Reply:
Or why do you specifically test for single files here?

Reply:
cause a single file should be rejected when a multiple drop is done with the multiple boolean set on false. As i understand it that's what is tested now.

To the first comment, i could change `expect(rejected.length).toEqual(1)` to `expect(rejected.length).toBeGreaterThanOrEqual(1)` then every case would be checked here.

Reply:
nevermind, i added a test case for rejected files on a multiple file drop if multiple=false